developer-docs-site/docs/issues-and-workarounds.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Issues and Workarounds"
+slug: "issues-and-workarounds"
+---
+
+# Issues and Workarounds
+
+This page documents issues that frequently come up and the suggested workarounds. 
+
+:::tip Help keep this page up-to-date
+If you found an issue that is not on this page, submit a [GitHub Issue](https://github.com/aptos-labs/aptos-core/issues). Make sure to follow the issue format used in this document. 
+:::
+
+## Nodes
+
+### Terraform "Connection Refused" error
+
+#### Description
+
+When running terraform, the command errors out with a connection refused error message.
+
+  ```
+  Error: Get "http://localhost/api/v1/namespaces/aptos": dial tcp 127.0.0.1:80: connect: connection refused
+  ```
+
+#### Workaround
+
+This likely means that the state of the install is out of sync with the saved terraform state file located in the storage bucket (configured during `terraform init` statement).  This could happen if the cluster or other components were deleted outside of terraform, or if terraform had an error and did not finish.  Use the following commands to check the state.  Delete the state that is related to the error message.  You will likely need to run terraform destroy, clean up the environment, and run the terraform script again.  
```

Comment:
How will they know which state is related to the error message so they may delete it?

Reply:
See below comment.

Reply:
Thanks, resolving.